packages/components/psammead-section-divider/src/index.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import styled from 'styled-components';
+import { oneOf, shape, string } from 'prop-types';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import {
+  GEL_GROUP_3_SCREEN_WIDTH_MIN, // 600px
```

Comment:
Im not a big fan of these comments as they technically could go out of sync with the package they reference. Can they be removed please? Lines 6->22